### ISSUE TITLE
faster_rcnn_inception step time increase during training

### ISSUE THREAD

**Issue by wuchichung** [tensorflow/models#4613]
I am using the faster_rcnn_inception_v2 model on Linux Ubuntu 16.04
TensorFlow installed from docker in version 1.5.0-gpu

The step time is increasing during training 

Initially:

INFO:tensorflow:global step 7442: loss = 0.2357 (0.135 sec/step)
INFO:tensorflow:global step 7442: loss = 0.2357 (0.135 sec/step)
INFO:tensorflow:global step 7443: loss = 0.3620 (0.140 sec/step)
INFO:tensorflow:global step 7443: loss = 0.3620 (0.140 sec/step)
INFO:tensorflow:global step 7444: loss = 0.3198 (0.145 sec/step)
INFO:tensorflow:global step 7444: loss = 0.3198 (0.145 sec/step)
INFO:tensorflow:global step 7445: loss = 0.3645 (0.142 sec/step)
INFO:tensorflow:global step 7445: loss = 0.3645 (0.142 sec/step)
INFO:tensorflow:global step 7446: loss = 0.4276 (0.152 sec/step)
INFO:tensorflow:global step 7446: loss = 0.4276 (0.152 sec/step)
INFO:tensorflow:global step 7447: loss = 0.3397 (0.141 sec/step)
INFO:tensorflow:global step 7447: loss = 0.3397 (0.141 sec/step)
INFO:tensorflow:global step 7448: loss = 0.2479 (0.141 sec/step)
INFO:tensorflow:global step 7448: loss = 0.2479 (0.141 sec/step)
INFO:tensorflow:global step 7449: loss = 0.4725 (0.132 sec/step)
INFO:tensorflow:global step 7449: loss = 0.4725 (0.132 sec/step)
INFO:tensorflow:global step 7450: loss = 0.2342 (0.145 sec/step)
INFO:tensorflow:global step 7450: loss = 0.2342 (0.145 sec/step)
INFO:tensorflow:global step 7451: loss = 0.3626 (0.144 sec/step)
INFO:tensorflow:global step 7451: loss = 0.3626 (0.144 sec/step)
INFO:tensorflow:global step 7452: loss = 0.2634 (0.139 sec/step)
INFO:tensorflow:global step 7452: loss = 0.2634 (0.139 sec/step)
INFO:tensorflow:global step 7453: loss = 0.2213 (0.142 sec/step)
INFO:tensorflow:global step 7453: loss = 0.2213 (0.142 sec/step)
INFO:tensorflow:global step 7454: loss = 0.1826 (0.139 sec/step)
INFO:tensorflow:global step 7454: loss = 0.1826 (0.139 sec/step)
INFO:tensorflow:global step 7455: loss = 0.3294 (0.139 sec/step)

However, several steps later 

INFO:tensorflow:global step 7644: loss = 0.4095 (3.916 sec/step)
INFO:tensorflow:global step 7645: loss = 0.1419 (3.705 sec/step)
INFO:tensorflow:global step 7645: loss = 0.1419 (3.705 sec/step)
INFO:tensorflow:global step 7646: loss = 0.5428 (3.817 sec/step)
INFO:tensorflow:global step 7646: loss = 0.5428 (3.817 sec/step)
INFO:tensorflow:global step 7647: loss = 0.4392 (3.413 sec/step)
INFO:tensorflow:global step 7647: loss = 0.4392 (3.413 sec/step)
INFO:tensorflow:global step 7648: loss = 0.1793 (3.816 sec/step)
INFO:tensorflow:global step 7648: loss = 0.1793 (3.816 sec/step)
INFO:tensorflow:global step 7649: loss = 0.2287 (3.414 sec/step)
INFO:tensorflow:global step 7649: loss = 0.2287 (3.414 sec/step)
INFO:tensorflow:global step 7650: loss = 0.6222 (3.716 sec/step)
INFO:tensorflow:global step 7650: loss = 0.6222 (3.716 sec/step)
INFO:tensorflow:global step 7651: loss = 0.2047 (3.013 sec/step)
INFO:tensorflow:global step 7651: loss = 0.2047 (3.013 sec/step)
INFO:tensorflow:global step 7652: loss = 0.2048 (3.311 sec/step)
INFO:tensorflow:global step 7652: loss = 0.2048 (3.311 sec/step)
INFO:tensorflow:global step 7653: loss = 0.3722 (3.223 sec/step)
INFO:tensorflow:global step 7653: loss = 0.3722 (3.223 sec/step)
INFO:tensorflow:global step 7654: loss = 0.3551 (3.905 sec/step)
INFO:tensorflow:global step 7654: loss = 0.3551 (3.905 sec/step)
INFO:tensorflow:global step 7655: loss = 0.0131 (4.021 sec/step)
INFO:tensorflow:global step 7655: loss = 0.0131 (4.021 sec/step)
INFO:tensorflow:global step 7656: loss = 0.4525 (3.316 sec/step)
INFO:tensorflow:global step 7656: loss = 0.4525 (3.316 sec/step)
INFO:tensorflow:global step 7657: loss = 0.3143 (4.006 sec/step)
INFO:tensorflow:global step 7657: loss = 0.3143 (4.006 sec/step)
INFO:tensorflow:global step 7658: loss = 0.3434 (3.514 sec/step)
INFO:tensorflow:global step 7658: loss = 0.3434 (3.514 sec/step)
INFO:tensorflow:global step 7659: loss = 0.7263 (3.314 sec/step)
INFO:tensorflow:global step 7659: loss = 0.7263 (3.314 sec/step)

I have tried to disable the summary writer to setting trainer's save_summaries_secs to None, but it does not help.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by wuchichung** [comment]
after updating the linux kernel from 4.4 to 4.13, this problems seem to be fixed.

**Comment by wt-huang** [comment]
Closing as this is resolved
